package Practice;

public class ExceptionProgram {

	public static void main(String[] args) {
		
		int a =10 , b= 0 ;
		
		int c=0;
		
		try
		{

		System.out.println(c=a/b);
		}
		catch(Exception e)
		{
			System.out.println("Exception are "+e);
		}
	

	}

}
